client/src/components/User/DiskUsage/Management/Cleanup/ReviewCleanupDialog.vue

```diff
@@ -0,0 +1,252 @@
+<template>
+    <b-modal id="review-cleanup-dialog" title-tag="h2" centered @show="onShowModal" v-model="showDialog" static>
+        <template v-slot:modal-title>
+            {{ title }}
+            <span class="text-primary h3">{{ totalItems }} items</span>
+        </template>
+        <div>
+            {{ captionText }}
+            <b>
+                <b-link @click="onSelectAllItems">select all {{ totalItems }} items</b-link>
```

Comment:
total items maxes out at 500 because of the API limit, there may be more items actually. Not sure how to convery this, but if we sort by size it's maybe not that important

Reply:
Oh, you are right, I didn't try with so many datasets. I guess we can set the `limit` query parameter in the requests to a pretty higher number like 10000 to bypass the 500 default limit but still there will be a limit.

The sorting by size seems to be working pretty well though :)